from exceptions.error import Error


class FormatError(Error):
    pass